src/mqueryfront/src/QueryField.js

```diff
@@ -12,13 +12,22 @@ class QueryField extends Component {
         });
 
         var multiselect = null;
+        var placeHolder = "everywhere";
```

Comment:
`placeholder` instead of `placeHolder` (for consistency with other places in code, also it's one word not two)